### PR TITLE
🔧(lti consumer) sample configuration for our configurable LTI consumer

### PR DESCRIPTION
**Pull request by rmoch**
# Purpose
This PR is a part of a group of changes to `edx-platform`, `fun-platform` and a new repository
`configurable-lti-consumer` which aims to allow CMS to propose several preconfigured instances
of `lti-consumer` block

# Proposal
This PR propose a working demonstration configuration

### CODE REVIEW

**Comment by jmaupetit** [comment]
Why did you choose to add settings only for development?

**Comment by sampaccoud** [comment]
@rmoch can you add descriptions to your PR so that people external to our internal discussions can follow our work?
The FUN best practice is 2 sections : Purpose and Proposal

**Comment by sampaccoud** [comment]
Is the plan to merge this to master or open a new branch: "oeplus-hawthorn.1" or some other cool name?

File: config/cms/docker_run_production.py [hunk 1]
```diff
@@ -727,6 +727,30 @@
...
+        }]
+    }]
+}
+
+# configurable-lti-consumer configuration
+CONFIGURABLE_XBLOCKS_SETTINGS = config(
+    "CONFIGURABLE_XBLOCKS_SETTINGS", default=DEFAULT_CONFIGURABLE_XBLOCKS_SETTINGS, formatter=json.loads
+)
+
+from configurable_lti_consumer import configurable_xblocks
```
Comment (sampaccoud) [inline config/cms/docker_run_production.py]: Maybe add comment to explain what this function does and why we need to override this setting?

File: config/cms/docker_run_production.py [hunk 2]
```diff
@@ -716,6 +716,33 @@
...
+
+# configurable-lti-consumer configuration
+CONFIGURABLE_XBLOCKS_SETTINGS = config(
+    "CONFIGURABLE_XBLOCKS_SETTINGS", default=DEFAULT_CONFIGURABLE_XBLOCKS_SETTINGS, formatter=json.loads
+)
+
+# Here we set XBLOCK_SELECT_FUNCTION setting to a configurable_lti_consumer
+# helper function that will be passed to XBock.load_class
+# method to filter multiple Python endpoints for the same xblock (lti_consumer)
+from configurable_lti_consumer import configurable_xblocks
```
Comment (sampaccoud) [inline config/cms/docker_run_production.py]: Maybe the import should be at the top of the file?

File: config/cms/docker_run_production.py [hunk 3]
```diff
@@ -384,7 +386,7 @@
 
 ################ SECURE AUTH ITEMS ###############################
 
-############### XBlock filesystem field config ##########
+    ############### XBlock filesystem field config ##########
```
Comment (jmaupetit) [inline config/cms/docker_run_production.py]: I think this indentation was not... intended :wink:

File: config/cms/docker_run_production.py [hunk 4]
```diff
@@ -597,6 +599,33 @@
     "ELASTIC_SEARCH_CONFIG", default=[{}], formatter=json.loads
 )
 
+################ XBLOCK RELATED ###############
```
Comment (jmaupetit) [inline config/cms/docker_run_production.py]: `s/XBLOCK RELATED/XBLOCKS CONFIGURATION/` WDYT?

**Comment by jmaupetit** [comment]
@rmoch I think you can squash the 3 settings-related commits and we will be good to merge this :clap:

**Pull request merged**